### PR TITLE
Add new boards (deneyapkart/1a/g)

### PR DESCRIPTION
**Pull request by ozgurbostan**
- Deneyap Kart
- Deneyap Kart 1A
- Deneyap Kart G

### CODE REVIEW

File: ports/espressif/boards/deneyap_kart_1a/sdkconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+CONFIG_SPIRAM=y
+CONFIG_SPIRAM_BOOT_INIT=y
+CONFIG_SPIRAM_IGNORE_NOTFOUND=y
+CONFIG_SPIRAM_USE_MEMMAP=y
+# CONFIG_SPIRAM_USE_CAPS_ALLOC is not set
+# CONFIG_SPIRAM_USE_MALLOC is not set
+CONFIG_SPIRAM_MEMTEST=y
+# CONFIG_SPIRAM_ALLOW_BSS_SEG_EXTERNAL_MEMORY is not set
+# CONFIG_SPIRAM_ALLOW_NOINIT_SEG_EXTERNAL_MEMORY is not set
+CONFIG_SPIRAM_CACHE_WORKAROUND=y
```
Comment (dhalbert) [inline ports/espressif/boards/deneyap_kart_1a/sdkconfig:19]: Missing newline at end of file here.

Comment (microdev1) [reply ports/espressif/boards/deneyap_kart_1a/sdkconfig:19]: pre-commit excludes the following but it seems **OK** to have newlines in these files so these can be removed

- `ports/espressif/esp-idf-config/.*`
- `ports/espressif/boards/.*/sdkconfig`

https://github.com/adafruit/circuitpython/blob/3c64b5212fe396eca4b52c26fd92d80e61deb7a8/.pre-commit-config.yaml#L11

Comment (dhalbert) [reply ports/espressif/boards/deneyap_kart_1a/sdkconfig:19]: @microdev1 Yeah, I'd agree with that certainly for the `sdkconfig` files. Do you know if the `.csv`s in `esp-idf-config/` can have blank lines also? I would hope so.

Comment (microdev1) [reply ports/espressif/boards/deneyap_kart_1a/sdkconfig:19]: The have newlines now so it is fine I think.

File: ports/espressif/boards/deneyap_kart/sdkconfig [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+CONFIG_SPIRAM=y
+CONFIG_SPIRAM_BOOT_INIT=y
+CONFIG_SPIRAM_IGNORE_NOTFOUND=y
+CONFIG_SPIRAM_USE_MEMMAP=y
+# CONFIG_SPIRAM_USE_CAPS_ALLOC is not set
+# CONFIG_SPIRAM_USE_MALLOC is not set
+CONFIG_SPIRAM_MEMTEST=y
+# CONFIG_SPIRAM_ALLOW_BSS_SEG_EXTERNAL_MEMORY is not set
+# CONFIG_SPIRAM_ALLOW_NOINIT_SEG_EXTERNAL_MEMORY is not set
+CONFIG_SPIRAM_CACHE_WORKAROUND=y
```
Comment (dhalbert) [inline ports/espressif/boards/deneyap_kart/sdkconfig:19]: missing newline at end of file

**Comment by dhalbert** [comment]
@deneyapkart I tried to fix the missing newlines myself, but you didn't give permission for repo maintainers to push commits to your PR's. So we'll wait for your changes.

**Comment by ozgurbostan** [comment]
> @ozgurbostan  I tried to fix the missing newlines myself, but you didn't give permission for repo maintainers to push commits to your PR's. So we'll wait for your changes.

@dhalbert I have just fixed the missing new line.

File: ports/espressif/boards/deneyap_kart_1a/sdkconfig (hunk 1 above)
Comment (dhalbert) [inline ports/espressif/boards/deneyap_kart_1a/sdkconfig:19]: @ozgurbostan This file also is missing a newline.

**Review by dhalbert — CHANGES_REQUESTED** [review]

**Comment by ozgurbostan** [comment]
> Now, two more steps:
> 1. PR to add your Creator ID and board IDS to https://github.com/creationid/creators
> 2. PR to add these boards to https://circuitpython.org. See https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website.

Thanks for your review and support. We have already applied for the IDs and are waiting for the process to finish.

**Pull request merged**